docs/docs/40-operator-guide/40-security/30-access-controls.md

```diff
@@ -123,13 +138,9 @@ api:
   oidc:
     # ... omitted for brevity ...
     admins:
-      claims:
-        groups:
-        - devops
+      claims: '{"groups":["devops"]}'
     viewer:
-      claims:
-        groups:
-        - developers
+      claims: '{"groups":["developers"]}'
```

Comment:
Now that we have `kargo-user` ServiceAccount, I recommend we update docs to use `kargo-user` instead of `kargo-viewer` since `kargo-user` is a more limited, and a safer default.

Reply:
There is `projectCreators` also.

Reply:
We can mention all four roles, but I don't think the examples need to extend as far as showing the "behind the scenes" part for all four.

Reply:
Should those examples extend to `admins`, `projectCreators`, and `users` only?

Reply:
imho, show all four in the snippet of `values.yaml`, but when we get to the bit that reads "Behind the scenes, the configuration above..." we don't need to show the annotations on all four of the affected ServiceAccounts. It seems like just showing one would get the point across that what you put in the `values.yaml` just makes its way to an annotation on some chart-managed SA. (We currently show two, and even that seems like it's unnecessary to get the point across.)

Reply:
https://github.com/akuity/kargo/pull/5002/commits/160e326b48bd0e94594597f197086a51e8766621